backend/test/services/LibraryService.test.ts

```diff
@@ -180,6 +180,74 @@ describe('LibraryService', () => {
     });
   });
 
+  describe('create', () => {
+    it('returns 201 status with populated location when provided correct headers and a FHIR Library', async () => {
+      await supertest(server.app)
+        .post('/4_0_1/Library')
+        .send({ resourceType: 'Library', status: 'draft' })
+        .set('content-type', 'application/json+fhir')
+        .expect(201)
+        .then(response => {
+          expect(response.headers.location).toBeDefined();
+        });
+    });
+  });
+
+  describe('update', () => {
+    it('returns 200 when provided correct headers and a FHIR Library whose id is in the database', async () => {
+      await createTestResource(
+        { resourceType: 'Library', type: { coding: [{ code: 'logic-library' }] }, id: 'exampleId', status: 'draft' },
+        'Library'
+      );
+      await supertest(server.app)
+        .put('/4_0_1/Library/exampleId')
+        .send({ resourceType: 'Library', id: 'exampleId', status: 'active' })
+        .set('content-type', 'application/json+fhir')
+        .expect(200);
+      await supertest(server.app)
+        .get('/4_0_1/Library/exampleId')
+        .expect(200)
+        .then(response => {
+          expect(response.body.id).toEqual('exampleId');
+        });
+    });
```

Comment:
I think this second supertest call isn't necessary here. Just ensuring that we get back a 200 response seems sufficient to me for this test.